src/constants.ts

```diff
@@ -110,7 +110,9 @@ export const PRINT_LOGO =
 export const PRINT_LOGO_TEXT = "Ministry of Health";
 
 export const MAIN_STORE_LOCATION_TAG = "Main Store";
-
+export const MAIN_PHARMACY_LOCATION_TAG = "Main Pharmacy";
+export const DISPENSARY_LOCATION_TAG = "Dispensary";
```

Comment:
should this read dispensary or dispensing?

Reply:
Infact this wasn't being used i have removed it and also the tags should be specific to how the implementation needs to define them